JsonRPC/wox.py

```diff
@@ -4,9 +4,9 @@
 import sys
 import inspect
 
-class Wox(object):
+class Flow.Launcher(object):
```

Comment:
I do not know a lot about python, but is it valid to use a dot in a class name?

Reply:
good pick up. will fix that up in the other PR.